internal/objects/validator.go

```diff
@@ -54,6 +54,15 @@ type request struct {
 	obj    *unstructured.Unstructured
 	oldObj *unstructured.Unstructured
 	op     k8sadm.Operation
+	gvr    metav1.GroupVersionResource
+}
+
+func (req request) gr() schema.GroupResource {
+	return schema.GroupResource{Group: req.gvr.Group, Resource: req.gvr.Resource}
+}
+
+func (req request) name() types.NamespacedName {
```

Comment:
Maybe just have this return a string rather than having to add `.String()` to every call?

Reply:
I prefer strongly typed functions, but I will fix. 😄

Reply:
I'd typically agree but for an unexported utility function, I prefer readability. If we were to export this, I'd also consider something more strongly typed.